### PR TITLE
0009694: Added size column in the server list

### PR DESCRIPTION
**Pull request by evgen1137**
This is an a little bit ugly realization of [#9694](https://bugs.mtasa.com/view.php?id=9694): adds a new column to the server list that displays the total size of all currently loaded resources on the server.
TODO: sorting in this column
Screenshot:
screenshot

### CODE REVIEW

**Comment by Disinterpreter** [comment]
anyone?

File: Client/core/Serverbrowser/CServerBrowser.cpp [hunk 1]
```diff
@@ -1075,12 +1121,16 @@ void CServerBrowser::AddServerToList ( const CServerListItem * pServer, const Se
         const SString strPing             = pServer->nPing == 9999 ? "" : SString ( "%d", pServer->nPing );
         const SString strPingSortKey      = SString ( "%04d-", pServer->nPing ) + pServer->strTieBreakSortKey;
 
+        const SString strResSize          = GetDataUnit(static_cast < unsigned long long > ( pServer->fResSize * 1024 ) );
+        const SString strResSizeSortKey   = SString ( "-", 0 ) + pServer->strTieBreakSortKey; // TODO
```
Comment (qaisjp) [inline Client/core/Serverbrowser/CServerBrowser.cpp:1125]: Rogue `TODO`?

**Comment by CrosRoad95** [comment]
I think, this could be in mta even show incorrect value ( due part of files is already downloaded )

**Comment by ccw808** [comment]
It should display the download size that will actually occur. Otherwise I think this feature will generate complaints

**Comment by qaisjp** [comment]
Thanks for your pull request! Unfortunately we can't merge this as it does not accurately show the correct download size.

**Pull request closed without merging**